lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1559,7 +1559,10 @@ sub _asset_find ($name, $type, $parents) {
 
 sub allocate_network ($self, $name) {
     my $vlan = $self->_find_network($name);
-    return $vlan if $vlan;
+    if ($vlan) {
+        $self->networks->find_or_create({name => $name, vlan => $vlan});
```

Comment:
The contents in braces and brackets may be separated by a space.

Reply:
I think it's possible to do that. But in general we have a tool called "perltidy" that is also running as part of the continuous integration test suite and ensures that we use a consistent style. So in short: As long as perltidy does not complain we can accept the code as is :)